Makefile

```diff
@@ -44,3 +44,29 @@ SHELL = bash
 .SECONDARY:
 .DELETE_ON_ERROR:
 .PHONY: FORCE
+
+preflight-dev-kubeconfig:
+	@if [ -z "$(DEV_KUBECONFIG)" ] ; then \
+		echo "DEV_KUBECONFIG must be set"; \
+		exit 1; \
+	fi
+.PHONY: preflight-dev-kubeconfig
+
+deploy: push preflight-cluster
+	$(MAKE) deploy-only
+.PHONY: deploy
+
+deploy-only: preflight-dev-kubeconfig
+	kubectl apply -f docs/yaml/aes-crds.yaml && \
+	kubectl wait --for condition=established --timeout=90s crd -lproduct=aes && \
```

Comment:
these two kubectl calls are missing --kubeconfig $(DEV_KUBECONFIG), but we can fix that on teh shared/emisary pr to master.